privacyidea/api/lib/prepolicy.py

```diff
@@ -1286,6 +1286,19 @@ def pushtoken_add_config(request, action):
         else:
             raise PolicyError("Missing enrollment policy for push token: {0!s}".format(PUSH_ACTION.FIREBASE_CONFIG))
 
+        # Get the sslverify definition from the policies
+        ssl_verify = g.policy_object.get_action_values(
+            action=PUSH_ACTION.SSL_VERIFY,
+            scope=SCOPE.ENROLL,
+            realm=token_realm,
+            user=token_user,
+            resolver=token_resolver,
+            client=g.client_ip,
+            audit_data=g.audit_object.audit_data,
+            unique=True
+        )
+        if len(ssl_verify) == 1:
+            request.all_data[PUSH_ACTION.SSL_VERIFY] = list(ssl_verify)[0]
```

Comment:
I haven't tested this, but in case no ``push_ssl_verify`` policy is defined, doesn't this allow the user to disable SSL verification by passing a ``push_ssl_verify=0`` parameter to ``/token/init``?
I think it would be nicer to only allow disabling SSL verification if a corresponding policy exists.

Reply:
ok, so we need to add an else-branch

~~~~python
else:
    request.all_data[PUSH_ACTION.SSL_VERIFY] = 1
~~~~
right?

Reply:
Yup, sounds good!

Reply:
done in f1ae0319

Reply:
Thanks!